antipodal_gripper_controller/src/antipodal_gripper_action_controller_parameters.yaml

```diff
@@ -0,0 +1,61 @@
+antipodal_gripper_action_controller:
+  action_monitor_rate: {
+    type: double,
+    default_value: 20.0,
+    description: "Hz",
+    validation: {
+      gt_eq: [ 0.1 ]
+    },
+  }
+  joint: {
+    type: string,
+    default_value: "",
+  }
+  goal_tolerance: {
+    type: double,
+    default_value: 0.01,
+    validation: {
+      gt_eq: [ 0.0 ]
+    },
+  }
+  max_effort: {
+    type: double,
+    default_value: 10.0,
+    description: "Default effort used for grasping (Newtons)",
+    validation: {
+      gt_eq: [ 0.0 ]
+    },
+  }
+  max_velocity: {
```

Comment:
Just so I understand, this max velocity in m/sec should be an API available in the hardware driver. We don't compute and limit it right? Or did I understand wrong?

Reply:
Yeah, basically the `max_velocity_interface` will be initialized to `max_velocity`. If the hardware does not support this, then it probably won't export the `max_velocity_interface` in the first place. And we do not limit this value, it is upp to the hardware driver to use this value.

Reply:
Got it. Make sense